### ISSUE TITLE
[game/tuner_shop] Midnight Tuner shop testdriving

### ISSUE THREAD

**Issue by sameerv95** [InZidiuZ/op-framework-issue-tracker#2376]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
Write a summary of what your suggestion is about.
Add Test drive option for Tuner employees and add repair station.
 
**Reason:**
Why would your suggestion be a good change to the framework?
Firstly regarding the test drive system at Tuner shop as people can test drive vehicles at PDM it assure a guy that that particular vehicle can perform like this. But at tuner shop people have no idea how will a car perform they can only see the modifications and the exhaust note of the cars. Adding test drive for the vehicles can bring a lot of change in tuner shop.

Secondly regarding adding a repair station at tuner shop because a tuner is a guy who make changes in engine and body, he brakes it molds it repairs it. So why cant he repairs the vehicle also? as they have the upmost knowledge of cars. Adding one repair station at the back office where nitro filling system is present will be a good option for both customers and tuners.

**Comment by itachi069** [comment]
I agree for the test drive it will be great for tunershop. But not sure if the repair thing will be a option.

**Comment by twooot** [comment]
Repair is not going to be added, testdriving maybe
